rhea/system/hwtypes.py

```diff
@@ -39,11 +39,13 @@ def __call__(self, val=0):
 
 def Signals(sigtype, num_sigs):
     """ Create a list of signals
-    Arguments:
-        sigtype (bool, intbv): The type to create a Signal from.
-        num_sigs (int): The number of signals to create in the list
-    Returns:
-        sigs: a list of signals all of type `sigtype`
+
+    | Arguments:
```

Comment:
I noticed that line breaks are not conserved in the conversion process.
See [this](http://stackoverflow.com/questions/7033239/how-to-preserve-line-breaks-when-generating-python-docs-using-sphinx) issue.

From your first link, it seems like activating napoleon would allow us to use google style docstrings and preserve formatting. Is that what we should do here?

This is from the current conf.py
`extensions = ['sphinx.ext.autodoc', 'sphinx.ext.doctest', 'sphinx.ext.intersphinx', 'sphinx.ext.todo', 'sphinx.ext.coverage', 'sphinx.ext.mathjax', 'sphinx.ext.viewcode']`

and would be extended by `'sphinxcontrib.napoleon'`.